library/src/doo/karma.clj

```diff
@@ -93,7 +94,9 @@
       "client" {"args" ["doo.runner.run_BANG_"]}
       "singleRun" true
       "plugins" (into ["karma-cljs-test"] launcher-plugins)}
-     (get-in opts [:karma :config]))))
+     (get-in opts [:karma :config])
+     (coverage/settings ->out-dir opts)
```

Comment:
Could you swap the order of these two lines? It'd be best to have user's `[:karma :config]` last so that they can override anything they want, if needed.